.github/workflows/template.yml

```
@@ -76,26 +74,46 @@ jobs:
           path: lesson
           fetch-depth: 0
 
-      - name: Determine the proper reference to use
-        id: styles-ref
+      - name: Sync lesson with carpentries/styles
+        working-directory: lesson
         run: |
-          if [[ -n "${{ github.event.pull_request.number }}" ]]; then
-            echo "::set-output name=ref::refs/pull/${{ github.event.pull_request.number }}/head"
+          if [[ -n "${{ github.event.pull_request.number }}" ]]
```

Adding grouping of log lines so that we can debug this a bit easier if things get hairy.

```suggestion
          echo "::group::Fetch Styles"
          if [[ -n "${{ github.event.pull_request.number }}" ]]
```